packages/agents/router/src/config.ts

```diff
@@ -218,14 +198,6 @@ export const getEnvConfig = (
         })(),
     };
 
-    const maxLag = chainConfig.subgraph?.maxLag ?? MIN_SUBGRAPH_SYNC_BUFFER;
```

Comment:
why are we removing max lag?

Reply:
oh are we just removing the subgraph config, i c

Reply:
Yep, it's not being used anywhere.